pinot-common/src/main/java/com/linkedin/pinot/common/utils/SegmentNameBuilder.java

```diff
@@ -40,7 +43,7 @@ public static String buildBasic(String tableName, String prefix) {
   public static class Realtime {
     public static final String ALL_PARTITIONS = "ALL";
 
-    public static String build(String groupId, String partitionRange, String sequenceNumber) {
+    public static String buildRealtimeV1Name(String groupId, String partitionRange, String sequenceNumber) {
```

Comment:
can we use high level/low level consume to name the methods. hard to keep track of what is V1 V2